src/main/scala/edu/ie3/powerFactory2psdm/main/RunConversion.scala

```diff
@@ -0,0 +1,13 @@
+package edu.ie3.powerFactory2psdm.main
+
+import com.typesafe.scalalogging.LazyLogging
+import edu.ie3.powerFactory2psdm.io.PfGridParser
+import edu.ie3.powerFactory2psdm.model.powerfactory.PowerFactoryGrid
+
+object RunConversion extends LazyLogging {
+
+  def main(args: Array[String]): Unit ={
```

Comment:
If you intend to provide this as a standalone application that is able to convert different grids into PSDM model structure, you might consider adding typesafe or any other config support. This would allow to distribute this application as binary. I'm fine without typesafe config support for now but if you consider it I would recommend adding an issue accordingly.

Reply:
Oh yes, will open an issue and definitely do that. Thanks for the input.